longest_common_prefix.py

```diff
@@ -0,0 +1,30 @@
+"""Write a function to find the longest common prefix string amongst an array
+of strings.
+
+If there is no common prefix, return an empty string "".
+ 
+
+Example 1:
+
+Input: strs = ["flower","flow","flight"]
+Output: "fl". """
+
+def longestCommonPrefix(strs: list[str]) -> str:
+	output = ""
+	for lists in range(len(strs[0])):
```

Comment:
when looping through a range, the convention is `idx` for "index". it's just an int, `lists` seems a little confusing